google-chart.html

```diff
@@ -279,39 +301,64 @@
         loadChartTypes: function() {
           this.chartTypes = {
             'area': google.visualization.AreaChart,
-            'bar': google.visualization.BarChart,
+            //'bar': google.visualization.BarChart,
             'bubble': google.visualization.BubbleChart,
             'candlestick': google.visualization.CandlestickChart,
-            'column': google.visualization.ColumnChart,
+            //'column': google.visualization.ColumnChart,
             'combo': google.visualization.ComboChart,
             'geo': google.visualization.GeoChart,
             'histogram': google.visualization.Histogram,
-            'line': google.visualization.LineChart,
+            //'line': google.visualization.LineChart,
             'pie': google.visualization.PieChart,
-            'scatter': google.visualization.ScatterChart,
+            //'scatter': google.visualization.ScatterChart,
             'stepped-area': google.visualization.SteppedAreaChart,
             'table': google.visualization.Table,
             'gauge': google.visualization.Gauge
           };
+
+          if (this.materialDesign){
+            this.chartTypes['bar']= google.charts.Bar;
+            this.chartTypes['column']= google.charts.Bar;
```

Comment:
spacing needs to be fixed throughout. Please see https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml